espnet2/asr/frontend/s3prl.py

```diff
@@ -36,6 +36,7 @@ def __init__(
         frontend_conf: Optional[dict] = get_default_kwargs(Frontend),
         download_dir: str = None,
         multilayer_feature: bool = False,
+        frontend_type = "s3prl"
```

Comment:
I don't think we need to pass the argument. Since it would be always the same, how about directly adding line 60 with "s3prl". The same for default one.

Reply:
ok